src/sentry/api/endpoints/group_notes.py

```diff
@@ -80,7 +80,7 @@ def post(self, request, group):
                 sentry_orgmember_set__organizationmemberteam__is_active=True,
                 is_active=True,
             ).exclude(id__in={u.id for u in actor_mentions.get('users')})
-            .values_list('id', flat=True)
+            .values_list('id', flat=True).distinct()
```

Comment:
`distinct()` shouldn't be needed here and doesn't actually change the result since `id` is already UNIQUE, you won't get duplicates.